tests/add_user.spec.ts

```diff
@@ -16,18 +17,24 @@ test.describe("Add User functional", () => {
     const addUserSteps = new AddUserSteps(page);
     const generator = new Generator();
 
+    const gender = generator.getGender();
     const username = generator.getName();
     const yearOfBirth = generator.getYearOfBirth();
-    const gender = generator.getGender();
+
+    const user = new UserDTO(gender, username, yearOfBirth);
```

Comment:
Move random user generation to the Generator class. Instead of calling generator for 3 separate fields it should be one call, that returns UserDto